rom/dev/README.md

```diff
@@ -97,22 +95,26 @@ Following are the main FUSE & Architectural Registers used by the Caliptra ROM f
-| FUSE_OWNER_PK_HASH              | 384          | Owner Public Key Hash                                   |
-| FUSE_FMC_SVN                    | 32           | FMC Security Version Number                             |
-| FUSE_RUNTIME_SVN                | 64           | Runtime Security Version Number                         |
+| FUSE_UDS_SEED                   | 384          | Obfuscated UDS                                          |
+| FUSE_FIELD_ENTROPY              | 384          | Obfuscated Field Entropy                                |
+| FUSE_KEY_MANIFEST_PK_HASH       | 384          | Hash of the four ECC and thiry-two LMS Manufacturer Public Keys   |
+| FUSE_KEY_MANIFEST_PK_HASH_MASK  | 32           | Manufacturer ECC Public Key Revocation Mask             |
+| FUSE_LMS_REVOCATION             | 32           | Manufacturer LMS Public Key Revocation Mask             |
+| FUSE_LMS_VERIFY                 | 32           | LMS Verification flag: <br> **0** - Verify Caliptra firmware images with ECDSA-only  <br> **1** - Verify Caliptra firmware images with both ECDSA and LMS |
+| FUSE_OWNER_PK_HASH              | 384          | Owner ECC and LMS Public Key Hash                       |
+| FUSE_FMC_KEY_MANIFEST_SVN       | 32           | FMC Security Version Number                             |
+| FUSE_RUNTIME_SVN                | 128          | Runtime Security Version Number                         |
 | FUSE_ANTI_ROLLBACK_DISABLE      | 1            | Disable SVN checking for FMC & Runtime when bit is set  |
-| FUSE_IDEVID_CERT_ATTR           | 768          | FUSE containing information for generating IDEVID CSR   |
-| CPTRA_DBG_MANUF_SERVICE_REG     | 32           | Manufacturing Services like IDEVID CSR upload           |
+| FUSE_IDEVID_CERT_ATTR           | 768          | FUSE containing information for generating IDEVID CSR  <br> **Word 0**: X509 Key Id Algorithm (2 bits) 1: SHA1, 2: SHA256, 2: SHA384, 3: Fuse <br> **Word 1,2,3,4,5**: Subject Key Id <br> **Words 7,8**: Unique Endpoint ID  |
+| CPTRA_DBG_MANUF_SERVICE_REG     | 16           | Manufacturing Services: <br> **Bit 0**: IDEVID CSR upload           |
 
 ## 7. Vaults
 
 Caliptra Hardware has the following vaults for storing various cryptographic material:
 
 1.	**Key Vault**: Used to store sensitive keys (Private Keys & CDI). Firmware cannot read or write the Key Vault content directly. Key Vault has 32 slots. Firmware can refer to the keys in the Key Vault via Slot numbers during cryptographic operations.
 2.	**PCR Bank**: Used to store measurements. PCR Bank has 32 PCRs. PCRs can be read or hash extended by the firmware. Direct write to PCR is not possible.
-3.	**Data Vault**: Data Vault contains a set of sticky (lock till next cold reset), non-sticky (lock till next update reset [CHECK] Warm reset?) and scratch registers.
+3.	**Data Vault**: Data Vault contains a set of sticky (lock till next cold reset), non-sticky (lock till next Warm reset) and scratch registers.
```

Comment:
We don't unlock on update reset? #930 is assuming that's the case...

Reply:
Updated text to reflect non-sticky registers get unlocked on both warm and update reset.